lib/usd/ui/layerEditor/mayaSessionState.cpp

```diff
@@ -59,19 +60,11 @@ MayaSessionState::~MayaSessionState()
     //
 }
 
-void MayaSessionState::setStage(PXR_NS::UsdStageRefPtr const& in_stage)
+void MayaSessionState::setStageEntry(StageEntry const& inEntry)
```

Comment:
I would like this method to continue being called setStage, it's still the current stage what we're setting. The "stage entry' is one in the stage list. Here it's how we're going to specify the stage, but we're not really setting 'the stage entry'.

Reply:
I'm confused on what you mean. The class logic changed, instead of holding a pointer to stage, which is ambiguous in the case where you have two proxy shapes with the same stage, we are holding a stageEntry which has the proxyshape path and the stage (and the name). So yes we are setting the CurrentStageEntry, naming it just setStage when clearly its setting the Whole struct seems more confusing in my opinion.